src/platform/platform_posix.c

```diff
@@ -441,7 +441,7 @@ CxPlatProcActiveCount(
     //
     return 1;
 #else
-    return (uint32_t)sysconf(_SC_NPROCESSORS_ONLN);
```

Comment:
It sounds like `_SC_NPROCESSORS_ONLN` might actually be the equivalent of "active" processors while `_SC_NPROCESSORS_CONF` is "max". Thoughts?

I also looked at the code and there are a couple of places that we call active where we should probably call max instead.